### ISSUE TITLE
Vid2Seq: Large-Scale Pretraining of a Visual Language Model for Dense Video Captioning

### ISSUE THREAD

**Issue by chaerinkong** [reyllama/paper-reviews#221]
# TL; DR

- Pose dense video captioning as seq2seq problem with *time tokens* to capture temporal correspondence.

# Motivation

- Video captioning requires both temporal localization of events and subsequent semantic captioning. Integrate this rather complicated pipeline with a unified seq2seq training.

# Formulation

<img width="1156" alt="image" src="https://github.com/reyllama/paper-reviews/assets/46308407/317012a3-04cb-4920-9529-f31c50d81b3c">

<img width="1093" alt="image" src="https://github.com/reyllama/paper-reviews/assets/46308407/39a5616f-acb7-4771-8e2d-d5e9273c9b04">

- Data: YT-Temporal-1B with transcribed speech (weak supervision GT)
- Architecture: Vision Encoder + Text Encoder + Text Decoder
- Pretrain: language modeling against transcribed speech + MLM on transcribed speech

# Evaluation

<img width="572" alt="image" src="https://github.com/reyllama/paper-reviews/assets/46308407/2cb37378-de0c-414c-aeba-5ab86a4591bc">
<img width="566" alt="image" src="https://github.com/reyllama/paper-reviews/assets/46308407/230cfd10-3eb6-43fc-9b9d-3cc042861bb7">
<img width="557" alt="image" src="https://github.com/reyllama/paper-reviews/assets/46308407/2408f59d-2137-41d3-bf6d-e3cd4dd34465">
<img width="1150" alt="image" src="https://github.com/reyllama/paper-reviews/assets/46308407/55a4d8d5-f700-4ac4-8566-41109d831df2">
<img width="479" alt="image" src="https://github.com/reyllama/paper-reviews/assets/46308407/5ffdaba3-7273-46ee-9379-6f8b2fb3e03b">

# Inspiration

- X2seq works well with large data.
